core/src/test/java/io/questdb/std/datetime/millitime/DateFormatCompilerTest.java

```diff
@@ -107,6 +107,22 @@ public void testDayOneDigit() throws Exception {
         assertThat("dyyyy", "2014-01-03T00:00:00.000Z", "32014");
     }
 
```

Comment:
Could you add more test cases  for each letter and test min, max values  and add similar tests to TimestampFormatCompilerTest ?

Reply:
I added more tests to both, please let me know if you think more tests are required

Reply:
Nice job ! Tests look good .